chart/portworx/templates/portworx-operator.yaml

```diff
@@ -4,24 +4,6 @@ metadata:
   name: portworx-operator
   namespace: {{ template "px.getDeploymentNamespace" . }}
 ---
-apiVersion: policy/v1beta1
```

Comment:
can we only remove this for v1.25 and above

Reply:
Installed catalog on v1.24 without PSP and all PX components came up fine.

Below output is from one node cluster, so only one pod of each component is running and rest is in `Pending` state.
```
kgpo -nvinayak-7 -w
NAME                                       READY   STATUS      RESTARTS   AGE
portworx-api-7frdx                         1/1     Running     0          21m
portworx-kvdb-vh8pb                        1/1     Running     0          15m
portworx-operator-6c9f6687bc-gwqq7         1/1     Running     0          22m
portworx-pvc-controller-5776d946bf-h5m88   0/1     Pending     0          21m
portworx-pvc-controller-5776d946bf-klz5c   0/1     Pending     0          21m
portworx-pvc-controller-5776d946bf-l5tnp   1/1     Running     0          21m
px-csi-ext-587d6d5548-h7vsb                5/5     Running     0          21m
px-csi-ext-587d6d5548-lf4fz                5/5     Running     0          21m
px-csi-ext-587d6d5548-xzmzs                5/5     Running     0          21m
px-hook-etcd-preinstall-xf7vq              0/1     Completed   0          22m
stork-5866f89d78-628kp                     1/1     Running     0          21m
stork-5866f89d78-6r5vn                     0/1     Pending     0          21m
stork-5866f89d78-p9gwh                     0/1     Pending     0          21m
stork-scheduler-cb678c5dd-ns8n4            1/1     Running     0          21m
stork-scheduler-cb678c5dd-ttbd6            0/1     Pending     0          21m
stork-scheduler-cb678c5dd-z2nms            0/1     Pending     0          21m
vinayak-7-szqsn                            2/2     Running     0          21m
```

```
k exec -it vinayak-7-szqsn -nvinayak-7 -- /opt/pwx/bin/pxctl status
Defaulting container name to portworx.
Use 'kubectl describe pod/vinayak-7-szqsn -n vinayak-7' to see all of the containers in this pod.
Status: PX is operational
Telemetry: Disabled or Unhealthy
Metering: Healthy
License: PX-Enterprise IBM Cloud (test) (expires in 444 days)
Node ID: 1651a34d-e1f1-43aa-a0ce-3da210fe169e
	IP: 10.240.64.53
 	Local Storage Pool: 1 pool
	POOL	IO_PRIORITY	RAID_LEVEL	USABLE	USED	STATUS	ZONE		REGION
	0	LOW		raid0		50 GiB	6.0 GiB	Online	us-south-2	us-south
	Local Storage Devices: 1 device
	Device	Path		Media Type		Size		Last-Scan
	0:1	/dev/vdd	STORAGE_MEDIUM_MAGNETIC	50 GiB		14 Oct 22 04:32 UTC
	total			-			50 GiB
	Cache Devices:
	 * No cache devices
	Kvdb Device:
	Device Path	Size
	/dev/vde	32 GiB
	 * Internal kvdb on this node is using this dedicated kvdb device to store its data.
Cluster Summary
	Cluster ID: vinayak-7
	Cluster UUID: f1c62a34-9efc-41cb-a2b3-30ed4d68dd0b
	Scheduler: kubernetes
	Nodes: 1 node(s) with storage (1 online)
	IP		ID					SchedulerNodeName	Auth		StorageNode	Used	Capacity	Status	StorageStatus	Version		Kernel			OS
	10.240.64.53	1651a34d-e1f1-43aa-a0ce-3da210fe169e	10.240.64.53		Disabled	Yes		6.0 GiB	50 GiB		Online	Up (This node)	2.12.0-e939b64	4.15.0-193-generic	Ubuntu 18.04.6 LTS
Global Storage Pool
	Total Used    	:  6.0 GiB
	Total Capacity	:  50 GiB
```